src/libs/Accessibility/index.ts

```diff
@@ -18,6 +18,15 @@ const useScreenReaderStatus = (): boolean => {
     return isScreenReaderEnabled;
 };
 
+const useReduceMotionStatus = (): boolean => {
+    const [isScreenReaderEnabled, setIsScreenReaderEnabled] = useState(false);
+    useEffect(() => {
+        AccessibilityInfo.isReduceMotionEnabled().then((enabled) => setIsScreenReaderEnabled(enabled));
+    }, []);
+
+    return isScreenReaderEnabled;
+};
```

Comment:
Let's use `useReducedMotion` from Reanimated instead (see [docs](https://docs.swmansion.com/react-native-reanimated/docs/device/useReducedMotion/) here).

Pros:
* It is synchronous (no need for additional render when promise is resolved and `false` changes to `true`)

Cons:
* it's not reactive (yet), meaning the setting state is read only once and you need to reset the app after you change the setting; however we plan to address it in the future